src/services/AbstractDFG.jl

```diff
@@ -971,6 +972,14 @@ function buildSubgraph(::Type{G},
     return destDFG
 end
 
+# Perhaps IIF.InMemDFGType should perhaps be incorporated as a DFG value, but this has been much debated in the past and hence just left as breadcrum.
```

Comment:
It doesn't really matter now since the only one is LightDFG. The main reason it was in iif was to help with testing and the transition between using GraphsDFG to LightDFG. We can make a DefaultDFGType const to use in case it ever changes in the future.